scripts/exec-sync.js

```diff
@@ -1,24 +1,29 @@
+// @ts-check
```

Comment:
These `exec` cleanup are fine, but maybe we can export `just-task-preset`'s exec here and improve on that one?

https://github.com/kenotron/just-task/blob/master/packages/just-task-preset/src/exec.ts

I've also invited you to be a collaborator there